packages/ui/react-ui-tabs/src/Tabs.stories.tsx

```diff
@@ -0,0 +1,74 @@
+//
+// Copyright 2024 DXOS.org
+//
+import React from 'react';
+
+import { Dialog } from '@dxos/react-ui';
+import { getSize } from '@dxos/react-ui-theme';
+import { withTheme } from '@dxos/storybook-utils';
+
+import { Tabs as NaturalTabs } from './Tabs';
+
+const content = {
+  t1: {
+    title: 'Rig Veda',
+    panel:
+      "Muse about Rig Veda finite but unbounded Drake Equation star stuff harvesting star light not a sunrise but a galaxyrise. Invent the universe bits of moving fluff courage of our questions from which we spring radio telescope kindling the energy hidden in matter. Globular star cluster another world globular star cluster emerged into consciousness invent the universe extraordinary claims require extraordinary evidence. A very small stage in a vast cosmic arena with pretty stories for which there's little good evidence two ghostly white figures in coveralls and helmets are softly dancing vanquish the impossible white dwarf across the centuries.",
```

Comment:
please use the existing lorem generators for consistency and leverage.
e.g., see react-ui-editor